app/uk/gov/hmrc/tariffclassificationfrontend/config/AppConfig.scala

```diff
@@ -47,12 +47,14 @@ class AppConfig @Inject()(val runModeConfiguration: Configuration, environment:
   lazy val emailRendererUrl: String = baseUrl("hmrc-email-renderer")
   lazy val fileStoreUrl: String = baseUrl("binding-tariff-filestore")
   lazy val decisionLifetimeYears: Int = getInt("app.decision-lifetime-years")
-  lazy val zoneId: ZoneId = ZoneId.of("UTC")
   lazy val fileUploadMaxSize: Int = loadConfig("fileupload.maxSize").toInt
   lazy val fileUploadMimeTypes: Set[String] = loadConfig("fileupload.mimeTypes").split(",").map(_.trim).toSet
   lazy val apiToken: String = loadConfig("auth.api-token")
   lazy val pdfGeneratorUrl: String = baseUrl("pdf-generator-service")
 
+  lazy val zoneId: ZoneId = ZoneId.of("UTC")
+  lazy val clock: Clock = Clock.system(zoneId)
```

Comment:
Isnt there a `Clock.systemUTC()`?

Reply:
You define `zoneId`, so I thought I wanted to use the same timezone in all code